openreview/matching.py

```diff
@@ -15,35 +14,49 @@ def get_conflicts(author_profiles, user_profile):
         authors_domain_conflicts.update(author_domain_conflicts)
         authors_relation_conflicts.update(author_relation_conflicts)
 
+    user_domain_conflicts, user_relation_conflicts = get_profile_conflicts(user_profile)
 
-	user_domain_conflicts, user_relation_conflicts = get_profile_conflicts(user_profile)
+    conflicts.update(authors_domain_conflicts.intersection(user_domain_conflicts))
+    conflicts.update(authors_relation_conflicts.intersection(user_relation_conflicts))
 
-	conflicts.update(authors_domain_conflicts.intersection(user_domain_conflicts))
-	conflicts.update(authors_relation_conflicts.intersection(user_relation_conflicts))
-
-	return list(conflicts)
+    return list(conflicts)
 
 def get_profile_conflicts(profile):
     domain_conflicts = set()
     relation_conflicts = set()
```

Comment:
maybe this "relation_conflicts" set should start with one item: the profile's id

relation_conflicts = set([profile.id])

what do you think?